src/include/win.h

```diff
@@ -383,7 +383,11 @@ extern int closedir(DIR *dir);
 
 /* pid related stuff */
 #define WNOHANG 1
+#ifndef LINKS_PYTHON
 #define pid_t	int
+#else
+#include "python.h"
+#endif
```

Comment:
All projects that link to python, try not to re-define pid_t and use "python.h"'s definition of pid_t instead. All projects that do not link to python, defind pid_t. pid_t is not available to earlier files but can be made available by including win.h. We can change it accordingly.